source/Conversation.cpp

```diff
@@ -555,7 +555,7 @@ bool Conversation::ElementIsValid(int node, int element) const
 
 
 // Parse the children of the given node to see if then contain any "gotos," or
-// "to shows." If so, link them up properly. Return true if gotos or
+// "to displays." If so, link them up properly. Return true if gotos or
```

Comment:
Tangential to this PR, but punctuation and `s`s might benefit from being outside the quotes for readability. Or, if it was written `"goto" or "to display" nodes`.

Reply:
Applied the second option.